sos/report/plugins/maas.py

```diff
@@ -56,18 +56,9 @@ def _remote_api_login(self):
 
         return ret['status'] == 0
 
-    def _is_snap_installed(self):
-        return self.exec_cmd('snap list maas')["status"] == 0
-
-    def check_enabled(self):
-        if super().check_enabled():
-            # deb-based MAAS and existing triggers
-            return True
-        # Do we have the snap installed?
-        return self._is_snap_installed()
-
     def setup(self):
-        self._is_snap = self._is_snap_installed()
+        self._is_snap = self.policy.package_manager.pkg_by_name(
+            'maas')['pkg_manager'] == 'snap'
```

Comment:
seems like the function here for `pkg_by_name('maas')` is coming back with `Null`, and hence causing the avocoda tests to fail.

Maybe, I better option is to re-instate the `_is_snap_installed()` function below

```python
    def _is_snap_installed(self):
        maas_pkg = self.policy.package_manager.pkg_by_name('maas')
        if maas_pkg:
            return maas_pkg['pkg_manager'] == 'snap'
        return False
```